examples/with-react-i18next/lib/withI18next.js

```diff
@@ -11,8 +11,12 @@ export const withI18next = (namespaces = ['common']) => ComposedComponent => {
       ? await ComposedComponent.getInitialProps(ctx)
       : {}
 
-    const i18nInitialProps =
-      ctx.req && !process.browser ? getInitialProps(ctx.req, namespaces) : {}
+    const i18nInitialProps = ctx.req && !process.browser 
```

Comment:
`ctx.req` is only defined on the server side, so you can remove `&& !process.browser`

Reply:
@timneutkens done